### ISSUE TITLE
Add support for commas in subscripts

### ISSUE THREAD

**Issue by balrobs** [mgreminger/EngineeringPaper.xyz#283]
While trying out this great software, I came across an important limitation for me .... I was unable to use commas in subscripts for variables.
![grafik](https://github.com/user-attachments/assets/64e8120f-d178-4196-b8d4-be9d689f2f78)
I am convinced that this would greatly improve the readability of documents.

Best wishes and congratulations for the great work

**Comment by mgreminger** [comment]
Thanks for the suggestion. Unfortunately, this usage of commas is not possible since commas in subscripts are used to index vectors and matrices as shown below (or in [this sheet](https://engineeringpaper.xyz/8pWM9yEqEPNntRBd6Jr9Sv)):
![image](https://github.com/user-attachments/assets/50f555ca-cc9c-4c0e-82aa-5f44bbdd7e6a)

**Comment by balrobs** [comment]
Thanks for the clarification,
it's a pity that the software can't distinguish between a variable subscript and a matrix index.
Would it then be possible as a workaround to allow some other similar alphanumerical character instead of the comma to somehow separate texts in variables subscripts? Maybe a semicolon or a blank space ... I noticed that even greek letters throw errors in variables subscripts.
Cheers

**Comment by mgreminger** [comment]
Yes, the lack of support for Greek characters in subscripts is a known issue (#246) and will be addressed at some point.

Unfortunately, supporting symbols in variable names (in the main variable or subscript) is not possible since the variable names need to map one-to-one to Python variable names and Python only allows underscores in variable names. Underscores are not easily supported since the underscore character is the keyboard shortcut to create a subscript.

Eventually, I plan to add support international characters in variable names, as [Python allows](https://www.asmeurer.com/python-unicode-variable-names/), but this may take a while to implement.

**Comment by balrobs** [comment]
> Eventually, I plan to add support international characters in variable names, as [Python allows](https://www.asmeurer.com/python-unicode-variable-names/), but this may take a while to implement.

That would be great!! "Good things take time"

> Unfortunately, supporting symbols in variable names (in the main variable or subscript) is not possible since the variable names need to map one-to-one to Python variable names and Python only allows underscores in variable names. Underscores are not easily supported since the underscore character is the keyboard shortcut to create a subscript.

Here I could imagine the following solution to be able to use the underscore character in the variables: change the keyboard shortcut to create subscripts from underscore to dot?
